task/rhacs-deployment-check/3.71/samples/pipeline.yaml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
Could we also add a sample for M2M usage?

Specifically also with the PVC for the shared workspace then which will hold the roxctl config for ease of use for some folks.

Reply:
I can add the m2m sample (pipeline + pipelinerun) here as well. So far it is only present in the m2m task.